auto_ncpt.ipynb

```diff
@@ -105,15 +106,165 @@
     "    layout=layout,\n",
     ")\n",
     "\n",
+    "\n",
+    "output_control = Text(\n",
+    "    value=\"./outputs/\",\n",
+    "    description=\"Outputs path\",\n",
+    "    placeholder=\"hf_xyz\",\n",
```

Comment:
change this, or remove it to avoid confusion

Reply:
fixed